hieradata_aws/class/integration/db_admin.yaml

```diff
@@ -1,16 +1,4 @@
 govuk_env_sync::tasks:
-  "pull_support-contacts_integration_daily":
-    ensure: "present"
```

Comment:
We'll need to mark this (and all other task removals) as `ensure: "absent"` so they are deleted from the machine, and then we can remove them.

Reply:
thanks Kevin, I've re-added these tasks and marked them as absent